src/log_to_disk.c

```diff
@@ -46,7 +46,7 @@
 
 pthread_mutex_t disk_mutex = PTHREAD_MUTEX_INITIALIZER;
 
-char lan_logline[81];
+char lan_logline[89];
```

Comment:
LOGLINELEN + 1 ? or would LOGLINELEN suffice?

Reply:
Hmm, comments say LOGLINELEN (87) includes all logline information including linefeed. That would mean LOGLINELEN+1 for buffer.
But as the newline is already stripped away one character less would be enough. 
Anyway we should use symbol instead of the hard coded number.

Reply:
Err, LOGLINELEN is defined as 88.

Reply:
Yes, log lines in the log file are 88 chars long including the NL. Internally it's the buffer length is the same as there is a NUL instead of NL.